website/server/models/challenge.js

```diff
@@ -149,7 +154,7 @@ schema.methods.syncToUser = async function syncChallengeToUser (user) {
     }
   });
 
-  toSave.push(user.save());
+  toSave.push(User.collection.update({_id: user._id}, user.toObject()));
   return Bluebird.all(toSave);
```

Comment:
That was what I had done initially, but then I had three statements for the user modifications in this file and then found that there are modifications in the server library that need to be saved to the user object as well so I thought this would be cleaner. 

Would you rather have the update statements for the improved performance?